frontend/src/types.ts

```diff
@@ -618,6 +618,7 @@ export interface FilterType {
     from_dashboard?: boolean
     funnel_step?: number
     funnel_viz_type?: string // this and the below param is used for funnels time to convert, it'll be updated soon
+    funnel_from_step?: number
```

Comment:
Maybe add a comment clarifying what this is for? And probably worth adjusting the comment above too